runtime/common/computationkind.go

```diff
@@ -57,7 +57,7 @@ const (
 	ComputationKindCreateArrayValue
 	ComputationKindTransferArrayValue
 	ComputationKindDestroyArrayValue
-	_
+	ComputationKindIterateArrayValue
```

Comment:
I added a new computation kind here. We could also use `ComputationKindLoop` because this is literally looping through the array. But introduced a new one in case we want to assign different weights to them.